### DISCUSSION TITLE
How to see a monthly reporting about net profit (Total Revenue - Internal rate)

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4151] [Questions & Answers]
I try to switch from toggl to Kimai. 
I've imported my first project and all it's timesheets to kimai, set the `hourly rates` and `internal rates` for the users. 

But only place where I see a tiny reporting for this "Internal-rates" is at "Customers Detail Page":

![image](https://github.com/kimai/kimai/assets/1778068/2a3fc1e5-c1a3-4fb4-8668-65b082a575da)

What I want to see is something like - `GROSS Profit / NET Profit` per: 
- Project & Month (or Date-Range)
- Project Total (over all Time) 
- User on 1 Project
- User on all Project he participate on
- User on Customers (he work for)

Is that possible in any way currently?

Those reportings are very important to see how many income per Customer, Project and User are archived in a given Time-Range.

## For Example:   
If you have employee A which got 70 EUR / hour and Employee B who got 90 EUR/hour. And customer pay 100 EUR / hour. 
Let's say A work 20 hours, B work 40 hours...  

```
This would be 
(100*60) - ((20*70)+(40*90)) = 1000.- NET profit, but 6000.- Gross Income (Revenue )
```

It would be very helpfully to easily see what NET Income the project produced for the date-range XXX - YYY. 

With the above data... let's assume I look at the last month of work, I would see 6000.- Gross Income, and 1000.- NET Income in a easy to read way

**Comment by xstable** [comment]
Still interested in an answer. Can anybody help?

**Reply by xstable** [reply]
👀

**Reply by suther** [reply]
Can anybody help with this?

**Comment by kevinpapst** [comment]
If you can't find it, it might not exist. 
If you have proposals, screenshots, mockups ... how that could look like: please share. 
But asking over and over again does not add any benefit to this discussion.

Btw: you can already fetch the data: 
- go to reports > monthly for all users > filter for "total revenue" > write down the total a
- switch to "internal price" and write down the total b
- subtract a - b
